### PR TITLE
fix(core): `Textfield` with `[content]` property has invalid behavior for `<input />`

### PR DESCRIPTION
**Pull request by nsbarsukov**

### CODE REVIEW

File: projects/demo-cypress/src/tests/textfield.cy.ts [hunk 1]
```diff
@@ -70,4 +77,34 @@ describe('Textfield', () => {
...
+            cy.mount(TestTextfield, {
+                componentProperties: {
+                    initialValue: '42',
+                    content: 'TOP-SECRET',
+                },
+            });
+        });
+
+        it('shows content for untouched field without focused', () => {
+            cy.get('tui-textfield').compareSnapshot('[content]-pristine-unfocused');
```
Comment (nsbarsukov) [inline projects/demo-cypress/src/tests/textfield.cy.ts:92]: **Before:**
textfield cy ts- content -pristine-unfocused

Comment (nsbarsukov) [reply projects/demo-cypress/src/tests/textfield.cy.ts:92]: **After:**
textfield cy ts- content -pristine-unfocused

File: projects/demo-cypress/src/tests/textfield.cy.ts [hunk 2]
```diff
@@ -70,4 +77,34 @@ describe('Textfield', () => {
...
+            cy.get('tui-textfield').compareSnapshot('[content]-focused');
+        });
+
+        it('shows content again after blur', () => {
+            cy.get('input[tuiTextfield]')
+                .focus()
+                .wait(300) // to ensure that all possible operations are finished
+                .blur();
+
+            cy.get('tui-textfield').compareSnapshot('[content]-after-blur');
```
Comment (nsbarsukov) [inline projects/demo-cypress/src/tests/textfield.cy.ts:107]: **Previous behavior:**
textfield cy ts- content -after-blur

Comment (nsbarsukov) [reply projects/demo-cypress/src/tests/textfield.cy.ts:107]: **New bahavior:**
textfield cy ts- content -after-blur

File: projects/demo-cypress/src/tests/textfield.cy.ts [hunk 3]
```diff
@@ -70,4 +77,34 @@ describe('Textfield', () => {
...
+        beforeEach(() => {
+            cy.mount(TestTextfield, {
+                componentProperties: {
+                    initialValue: '42',
+                    content: 'TOP-SECRET',
+                },
+            });
+        });
+
+        it('shows content for untouched field without focused', () => {
```
Comment (nsbarsukov) [inline projects/demo-cypress/src/tests/textfield.cy.ts:91]:
```suggestion
        it('shows content for untouched field without focus', () => {
```

**Review by splincode — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

**Pull request merged**